src/module/userRoleExtension/helper.js

```diff
@@ -89,11 +89,12 @@ module.exports = class userRoleExtensionHelper {
 	 * @param {Array<String>} projection - The fields to include in the returned documents.
 	 * @returns {Promise<Object>} - A promise that resolves with the found user role extensions or rejects with an error.
 	 */
-	static find(bodyQuery, projection) {
+	static find(bodyQuery, projection, limit, offset) {
 		return new Promise(async (resolve, reject) => {
 			try {
 				// Fetch user role extensions based on the provided query and projection
-				const result = await userRoleExtensionQueries.userDocuments(bodyQuery, projection)
+				const userRoles = await userRoleExtensionQueries.userDocuments(bodyQuery, projection, limit, offset)
```

Comment:
@adithyadinesh0412 check if this is optional. make sure existing usage of this api is not affected